OpenLIFUData/Resources/UI/OpenLIFUData.ui

```diff
@@ -26,12 +26,12 @@
       </item>
       <item>
        <widget class="ctkPathLineEdit" name="databaseDirectoryLineEdit">
+        <property name="filters">
+         <set>ctkPathLineEdit::Dirs</set>
```

Comment:
I see you are turning the `<string>` tag back to `<set>`

See 97c134dc1407e7ac9c8aa4557ebad9fb6b92a7c2 where I changed it to `<string>`

I'm not fully sure what's going on with this, maybe it's automatically changed by your version of Qt designer? My version of Qt designer seems to hate the `<set>` and ends up automatically dropping the important `ctkPathLineEdit::Dirs` ... not sure what is up with that. Did you Qt designer complain at all about `<string>` or did it just silently modify the tag?

Reply:
My Qt designer must have automatically modified this cause I didn't explicitly edit this or see any errors. I'll change it back to '<string>' for now and keep an eye out in future updates to see if this happens again.